go/cmd/relations/upstream_test.go

```diff
@@ -0,0 +1,245 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"slices"
+	"testing"
+
+	"cloud.google.com/go/datastore"
+	"github.com/google/osv.dev/go/osv/models"
+	"github.com/google/osv.dev/go/testutils"
+)
+
+func TestComputeUpstream(t *testing.T) {
+	bugs := map[string][]string{
```

Comment:
We probably should start using vulns as the term in the go version over bugs

Reply:
I had been doing that in most places, just missed it in a couple
Fixed